### ISSUE TITLE
How to see the training result

### ISSUE THREAD

**Issue by HURB** [jwyang/faster-rcnn.pytorch#313]
How could I see the training result such as ap,map and recall. Thank you!

**Comment by kentaroy47** [comment]
take a look at #test@readme.md

Test
If you want to evlauate the detection performance of a pre-trained vgg16 model on pascal_voc test set, simply run

python test_net.py --dataset pascal_voc --net vgg16 \
                   --checksession $SESSION --checkepoch $EPOCH --checkpoint $CHECKPOINT \
                   --cuda
Specify the specific model session, chechepoch and checkpoint, e.g., SESSION=1, EPOCH=6, CHECKPOINT=416.
